scripts/commands/reloadzone.lua

```diff
@@ -0,0 +1,21 @@
+---------------------------------------------------------------------------------------------------
+-- func: reloadzone
+-- desc: Attempt to reload specified zone lua without a restart.
+---------------------------------------------------------------------------------------------------
+
+cmdprops =
+{
+    permission = 4,
+    parameters = "ss"
+}
+
+function onTrigger(player, zoneLua, other)
+    if zoneLua and other then
+        local String = table.concat({"scripts/zones/", zoneLua})
+        package.loaded[String] = nil
+        require(String)
+        player:PrintToPlayer(string.format("Lua file '%s' has been reloaded.", String))
+    else
+        player:PrintToPlayer("Must Specify a [zone]/file lua file.")
+    end
+end
```

Comment:
I'll be honest, I am extremely new to C++ so I would approach doing something like this in the future. Otherwise I'll leave in these reload commands unless you don't want them added to LSB.